src/libstd/sys/cloudabi/mod.rs

```diff
@@ -1,20 +1,18 @@
 use crate::io::ErrorKind;
 use crate::mem;
 
+pub mod abi;
```

Comment:
We now need access to the abi types in libstd, but outside of this module. `parking_lot` requires it for the parking. And we don't depend on the `cloudabi` crate in libstd, so we get the types from here when built as a module inside libstd. Ping @EdSchouten.

Reply:
Using the copy in src/libstd/sys/cloudabi/abi/* seems fair.